explainaboard/metrics/metric.py

```diff
@@ -402,6 +425,17 @@ def calc_stats_from_data(
         """
         ...
 
+    @final
+    def get_seed(self) -> np.random.SeedSequence:
+        """Gets a numpy SeedSequence.
+
+        See Metric for the usage.
```

Comment:
Details should be written here as well.

Reply:
Added more explanations to the docstring.